distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -709,7 +709,7 @@ type Float
 
             2.5 . round rounding_mode=Rounding_Mode.Bankers == 2
     round : Integer -> Rounding_Mode -> Integer | Float ! Illegal_Argument
-    round self (decimal_places:Integer=0) (rounding_mode : Rounding_Mode = ..Half_Up) -> Float =
+    round self (decimal_places:Integer=0) (rounding_mode : Rounding_Mode = ..Half_Up) -> Integer | Float ! Illegal_Argument =
```

Comment:
I assume this was not failing because we have a conversion `Float.from Integer`.

Shall we add some test that asserts that `2.5.round . should_be_a Integer`?